docs: Call out in the CHANGELOG the format changes of the status logs decorations (#8174)
osquery/osquery: CHANGELOG.md

One potential breaking change, is in how [the watchdog calculates CPU utilization](https://github.com/osquery/osquery/pull/8104).
Previously, this calculation was based on physical CPUs, now it is based on virtual cores. We believe this makes more sense with modern CPUs.

> A second potential breaking change, is in PR [#8102](https://github.com/osquery/osquery/pull/8102), which beyond adding support to top level decorations for status logs when using the `--decorations_top_level` flag, also uniforms the status log decorations format to the results log.
> This in practice means that the `unixTime`, `severity` and `line` JSON fields are now numbers instead of strings.

Reviewer: ```suggestion
A second potential breaking change, is in PR [#8102](https://github.com/osquery/osquery/pull/8102). In addition to to adding decorations to the top level of the status logs, this PR normalizes the decorations format to the results log. In practice, this means that the `unixTime`, `severity` and `line` JSON fields are now numbers instead of strings.
```

Author: That flows better, but the "In addition to adding decorations to the top level of the status logs" is not "correct", in the sense that it's not something always like that, you need to enable it.
What about "In addition to adding the ability of having decorations to the top level of the status logs"

Reviewer: Maybe "In addition to allowing decorations to the top level of the status logs" ?

Reviewer: Or "enabling"

Author: > Or "enabling"

That would mean to me that they are now enabled (top level decorations); maybe `enabling to have`, because again we provide the ability to enable that behavior, it's not on by default.

Reviewer: Maybe? "enabling" and "allowing" both convey there's an option there. Though I think you're right, that "enabling" indicates it defaults to on, while "allowing"  implies the creation of an option
